schema/api.py

```
@@ -228,22 +238,24 @@ def get(self, cve_id):
-                        try:
-                            g.kill(block=False)
-                        except Exception:
-                            pass
-                return self.handle_error("Upstream timeout", 504)
+                if not all(g.ready() for g in greenlets):
+                    for g in greenlets:
+                        if not g.ready():
+                            try:
+                                g.kill(block=False)
+                            except Exception:
+                                pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)